src/test/java/de/tum/in/www1/artemis/util/ModelFactory.java

```diff
@@ -677,6 +677,36 @@ public static Exam generateExam(Course course) {
         return generateExamHelper(course, false);
     }
 
+    /**
+     * Generates an exam
+     *
+     * @param course      the associated course
+     * @param visibleDate the visible date of the exam
+     * @param startDate   the start date of the exam
+     * @param endDate     the end date of the exam
+     * @param testExam    if the exam is a test exam
+     * @return the created exam
+     */
+    public static Exam generateExam(Course course, ZonedDateTime visibleDate, ZonedDateTime startDate, ZonedDateTime endDate, boolean testExam) {
+        Exam exam = new Exam();
+        exam.setTitle((testExam ? "Test" : "Real") + " exam 1");
```

Comment:
```suggestion
        if (testExam) {
            exam.setTitle("Test exam 1");
        } else {
            exam.setTitle("Real exam 1");
        }
```

As in the teamscale comment, maybe this could be an `if` instead

Reply:
I personally think a ternary operator is better readable here, which is why I decided to keep it.